streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SavePointServiceImpl.java

```diff
@@ -333,6 +341,8 @@ private void handleSavepointResponseFuture(
               if (savepointResponse != null && savepointResponse.savePointDir() != null) {
                 applicationLog.setSuccess(true);
                 String savePointDir = savepointResponse.savePointDir();
+                savePoint.setPath(savePointDir);
```

Comment:
I think it'd be better to move lines [321](https://github.com/apache/incubator-streampark/pull/3493/files#diff-78be6d87725e2bef43b9f12e085bd59157e55cf33f79c0426c5862ebeae7711cR321)-[326](https://github.com/apache/incubator-streampark/pull/3493/files#diff-78be6d87725e2bef43b9f12e085bd59157e55cf33f79c0426c5862ebeae7711cR326) to line [344](https://github.com/apache/incubator-streampark/pull/3493/files#diff-78be6d87725e2bef43b9f12e085bd59157e55cf33f79c0426c5862ebeae7711cR344). 😊

Reply:
Done. Thank you for review.